### PR TITLE
Clean traces api format

### PR DESCRIPTION
**Pull request by schoren**
This PR removes the otel protobuf format mappings from the openapi spec, and proposes a new structure for tests, assertions, etc. This change has a really big impact both in the back and frontend, since it refactors most of the key models.

This is not functional, nor does it compile, but should be enough to discuss if the format is good enough.

## Checklist

- [x] tested locally
- [x] added new dependencies
- [ ] updated the docs
- [x] added a test

### CODE REVIEW

**Review by mathnogueira — COMMENTED** [review]
For me, those changes make sense.

File: api/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        format: uuid
+        readOnly: true
+      testId:
+        type: string
+        format: uuid
+        readOnly: true
+      traceId:
+        type: string
+        readOnly: true
+      spanId:
```
Comment (kdhamric) [inline api/tests.yaml:79]: What is spanId in reference to a TestRun?

Comment (schoren) [reply api/tests.yaml:79]: both TraceID and SpanID are generated by us and sent in the triggering transaction

**Review by mathnogueira — APPROVED** [review]
integration test passing is a big green light for this change :D

I reviewed everything and I don't have anything against any change, so I'll approve it.

**Review by xoscar — APPROVED** [review]
Great job @schoren! 🔥

**Pull request merged**